cmd/add.go

```diff
@@ -216,6 +237,14 @@ func init() {
 		"pretend to be the this user (only works if you started the server)")
 	addCmd.Flags().StringVar(&setMetadata, "metadata", "",
 		"key=val;key=val metadata to apply to all files in the set")
+	addCmd.Flags().Var(&setReason, "reason",
+		"storage reason: 'backup' | 'archive' | 'quarantine'")
+	addCmd.Flags().StringVar(&setReview, "review", "",
+		"months/years until review date, provided in format: <number><unit>, e.g. 1y for 1 year. "+
```

Comment:
time until review date (<number><y|m>, eg. 1y for 1 year), or exact review date in the format YYYY-MM-DD

Reply:
Likewise for remove